src/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ if(NOT ${CMAKE_SYSTEM_NAME} STREQUAL "Emscripten")
     )
 endif()
 
-target_link_libraries(mozillavpn PRIVATE glean lottie nebula translations)
+target_link_libraries(mozillavpn PRIVATE glean vpnglean lottie nebula translations)
```

Comment:
As some thoughts for a future PR, why not merge `glean` and `vpnglean` into one library?

Reply:
So, the idea is to have both and leave vpnglean behind a feature flag untill it's fully implemented / validated. It's not pretty and `glean` and `vpnglean` is probably the worst terminology choice... Got any better naming ideas?

Reply:
Hm, upon reading this again today I think I misunderstood what you said :sweat_smile:

Reply:
Well, since we don't change the build based on feature flags, we are always linking the glean.rs FFI library, so we might as well just bundle all of the "glean stuff" together into a single library as far as the `mozillavpn` target is concerned. For example, the `glean` library can depend on an internally-built glean.rs FFI library and export it as a public dependency.

Something along the lines of:
```
add_custom_command(
    OUTPUT ${CMAKE_CURRENT_BUILD_DIR}/libglean_ffi.a
    COMMAND cargo build ...
)
target_link_libraries(glean PUBLIC ${CMAKE_CURRENT_BUILD_DIR}/libglean_ffi.a)
```